definitions/checks/backup/incremental_parent_type.rb

```diff
@@ -0,0 +1,31 @@
+module Checks::Backup
+  class IncrementalParentType < ForemanMaintain::Check
+    metadata do
+      description 'Check if the incremental backup has the right type'
+      tags :backup
+      param :incremental_dir, 'Path to existing backup directory'
+      param :online_backup, 'Select for online backup', :flag => true, :default => false
+      param :sql_tar, 'Will backup include PostgreSQL tarball', :flag => true, :default => false
+      manual_detection
+    end
+
+    def run
+      return unless @incremental_dir
+
+      backup = ForemanMaintain::Utils::Backup.new(@incremental_dir)
+
+      existing_type = backup.online_backup? ? 'online' : 'offline'
+      new_type = @online_backup ? 'online' : 'offline'
+      msg = "The existing backup is an #{existing_type} one, but an #{new_type} one was requested."
```

Comment:
```suggestion
      msg = "The existing backup is an #{existing_type} backup, but an #{new_type} backup was requested."
```

Reply:
three times "backup" in one sentence?

Reply:
Just don't say it in front of a mirror. Better to be explicit?